README.md

```diff
@@ -8,52 +8,60 @@
 
 An example implementation of the [NLP Sandbox Date Annotator].
 
-## Specification
+## Overview
 
-- Implements the [NLP Sandbox Date Annotator OpenAPI specification]
-- Annotates date strings in clinical notes using simple regular expressions
+This repository provides a Python-Flask implementation of the [NLP Sandbox Date
+Annotator]. The Date Annotator is one of the first NLP Tools that can be
+benchmarked on nlpsandbox.io. A Date Annotator takes as input one clinical notes
+as well as information about the patient and outputs a list of predicted date
+annotations found in the clinical note.
 
-NLP Sandbox API implemented:
+- Date Annotator API version: 0.3.1
```

Comment:
Maybe include a header for these specs? e.g.

---

#### Specification
- Date Annotator API version: 0.3.1
- Tool version: 0.3.2
- Docker image: [nlpsandbox/date-annotator-example]

Reply:
I added the header